app/CocoaPods/Supporting Files/Info.plist

```diff
@@ -84,5 +84,10 @@
 	<string>NSApplication</string>
 	<key>SUFeedURL</key>
 	<string>https://app.cocoapods.org/sparkle.xml</string>
+	<key>NSAppTransportSecurity</key>
```

Comment:
Anyone else following this on GitHub also won't have context from #the-hivemind. I think the justification should be clear from the PR.